Makefile

```diff
@@ -159,7 +159,15 @@ ARTIFACTS ?= $(ROOT_DIR)/_artifacts
 E2E_CONF_FILE ?= $(ROOT_DIR)/test/e2e/config/e2e_conf.yaml
 E2E_OUT_DIR ?= $(ROOT_DIR)/test/e2e/_out
 E2E_CONF_FILE_ENVSUBST ?= $(E2E_OUT_DIR)/$(notdir $(E2E_CONF_FILE))
-E2E_CONTAINERS ?= quay.io/metal3-io/cluster-api-provider-metal3 quay.io/metal3-io/baremetal-operator quay.io/metal3-io/ip-address-manager
+
+CONTAINER_REGISTRY ?= quay.io
+CAPM3RELEASE ?= latest
```

Comment:
Group the tag variables and image variables. Makes this more readable.

We also need to cherry-pick this to release branches to have any effect there.